### PR TITLE
Modified Examples to work without systemjs

### LINKED ISSUE #9784: Can we have a simple example without system.js?
Is there not enough PDF.JS to learn without trying to work out the System.JS stuff?

```
// In production, the bundled pdf.js shall be used instead of SystemJS.
Promise.all([System.import('pdfjs/display/api'),
             System.import('pdfjs/display/worker_options'),
             System.import('pdfjs/display/network'),
             System.resolve('pdfjs/worker_loader')])
       .then(function (modules) {
  var api = modules[0];
  var GlobalWorkerOptions = modules[1].GlobalWorkerOptions;
  var network = modules[2];
```

### PR DESCRIPTION
**Pull request by RonLek**
Closes #9784 

@timvandermeij  @Snuffleupagus  Please review for changes, if any. Thanks!

### CODE REVIEW

**Review by Snuffleupagus — CHANGES_REQUESTED** [review]

**Comment by RonLek** [comment]
@Snuffleupagus Thanks for reviewing. Made the changes!

File: README.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ You can play with the PDF.js API directly from your browser using the live demos
 
 The repository contains a hello world example that you can run locally:
 
-+ [examples/helloworld/](https://github.com/mozilla/pdf.js/blob/master/examples/helloworld/)
++ [examples/learning/helloworld](https://github.com/mozilla/pdf.js/blob/master/examples/learning/helloworld.html)
```
[the remark this answers is not in the document]
Comment (timvandermeij) [reply README.md]: Yes, I fully agree with this.

File: README.md [hunk 2]
```diff
@@ -102,10 +102,6 @@ You can play with the PDF.js API directly from your browser using the live demos
 
 + [Interactive examples](http://mozilla.github.io/pdf.js/examples/index.html#interactive-examples)
 
-The repository contains a hello world example that you can run locally:
-
-+ [examples/helloworld/](https://github.com/mozilla/pdf.js/blob/master/examples/helloworld/)
-
 More examples can be found in the examples folder. Some of them are using the pdfjs-dist package, which can be built and installed in this repo directory via `gulp dist-install` command.
```
Comment (Snuffleupagus) [inline README.md]: Nit: Maybe it wouldn't be a bad idea to also make "examples folder" text into a link, i.e. change it to
```
[examples folder](https://github.com/mozilla/pdf.js/tree/master/examples/)
```

Comment (Snuffleupagus) [inline README.md]: Nit: Maybe it wouldn't be a bad idea to change the "examples folder" text into a link, i.e. like so:
[examples folder](https://github.com/mozilla/pdf.js/tree/master/examples/)
```

Comment (Snuffleupagus) [inline README.md]: Nit: Maybe it wouldn't be a bad idea to change the "examples folder" text into a link, i.e. like so:
```
[examples folder](https://github.com/mozilla/pdf.js/tree/master/examples/)
```

**Comment by RonLek** [comment]
And Done!!

**Review by Snuffleupagus — APPROVED** [review]

**Review by timvandermeij — APPROVED** [review]

**Comment by timvandermeij** [comment]
Nice work; thank you for your contribution!

**Pull request merged**